_releases/2023-04-12-1.8.0-released.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Crystal 1.8.0 is released!
+version: 1.8.0
+summary:
+thumbnail: +
+author:
+---
+
+We are delivering a new release with several bugfixes and improvements. Below we list the most important or interesting changes, without mentioning several bugfixes and smaller enhancements. For more details, visit the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.8.0). Breaking changes are marked with ⚠️.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.8.0) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+## Stats
+
+In this release we included [XXX changes since the 1.7.3 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.8.0) by XX contributors. We thank all the contributors for all the effort put into improving the language! ❤️
+
+Below we list the most remarkable changes in the language, compiler and stdlib.
+
+## Regex: PCRE2
+
+⚠️ A big and [necessary change](/2023/03/02/crystal-is-upgrading-its-regex-engine) in this release is that PCRE2 becomes the default engine for `Regex` ([#12978](https://github.com/crystal-lang/crystal/pull/12978)).
+Support for PCRE2 was added in 1.7 and stabilized in 1.7.3. We feel it's safe to use it by default.
+A fallback to PCRE is still possible, either automatically if PCRE2 is unavailable at compile time, or via the compiler flag `-Duse_pcre`.
+
+> **NOTE:** Package maintainers
+> From 1.8 onwards the Crystal compiler should be built with PCRE2 to ensure consistency.
+
+⚠️ As a measure to ease migration to PCRE2, `Regex::Options` gained more named members which work in both engine versions. To accomodate the additional values, the enum base type changes from `Int32` to `UInt64` ([#13223](https://github.com/crystal-lang/crystal/pull/13223)).
```

Comment:
```suggestion
⚠️ As a measure to ease migration to PCRE2, `Regex::Options` gained more named members which work in both engine versions. To accommodate the additional values, the enum base type changes from `Int32` to `UInt64` ([#13223](https://github.com/crystal-lang/crystal/pull/13223)).
```

Also might want to mention the existence of the `Regex::CompileOptions` alias here first, not just in the next section